src/components/FooterV2.stories.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import FooterV2 from './FooterV2'
+export default {
+  title: 'Components/FooterV2',
+  component: FooterV2,
+}
+
+const Template = ({ onBeforeInstall }) => {
+  return <FooterV2 onBeforeInstall={onBeforeInstall} />
+}
+export const standard = Template.bind({})
```

Comment:
Please add a mobile canvas and wide (monitor width) canvas so we can rely on UI reviews/tests on Chromatic.